src/modules/event.module.ts

```diff
@@ -1,13 +1,19 @@
 import { MicrobootstrapSettings, MicrobootstrapLoader } from '@hugobyte/microbootstrap';
 import { Container } from 'typedi';
 import { EventController } from '@controllers/index';
+import { util } from 'config';
+import { EventService } from '@services/index';
+import openwhisk from 'openwhisk';
 
 /**
  * Event Module initializes event controller
  */
 
 export const EventModule: MicrobootstrapLoader = (frameworkSettings: MicrobootstrapSettings | undefined) => {
     if (frameworkSettings) {
+        const { openwhiskApiHost, openwhiskApiKey, openwhiskNamespace } = util.loadFileConfigs();
```

Comment:
change method name to 
```suggestion
        const { openwhiskApiHost, openwhiskApiKey, openwhiskNamespace } = util.loadFileConfigurations();
```

Reply:
loadFileConfigs is a method from external library